src/assets/docs/search.md

```diff
@@ -0,0 +1,25 @@
+# Search
+
+Dockstore now offers faceted search, which allows for flexible querying of tools and workflows. Tabs are used to split up the results between tools and workflows. You can search for basic terms/phrases, filter using facets, and also used advanced search queries. Note that using advanced search after setting a basic search will override your basic search, and vice versa.
+
+## Filtering Your Search
+### Basic Search
+If you want to do a basic search for an exact keyword or phrase, use the search box present on the left side of the page. This will search all entries for the exact string you type.
```

Comment:
Hmm, I think it actually functions as a "one of" the words rather than an exact string.

Reply:
Is it better to say something like "If you want to do a basic search for a keyword or a collection of keywords, use the search box present on the left side of the page. This will search for matches that contain at least one of the keywords. Keywords should be separated  by spaces."

Reply:
Sounds good